install/migrations/update_9.2.0_to_9.2.1.php

```diff
@@ -441,14 +441,17 @@ function update920to921()
             'operatingsystemarchitectures_id'   => "0",
             'operatingsystemkernelversions_id'  => "0",
             'operatingsystemeditions_id'        => "0",
-            'OR' => [
-                ['license_number' => null],
-                ['license_number' => ""]
+            [
+                'OR' => [
+                    ['license_number' => null],
+                    ['license_number' => ""]
+                ]
             ],
-            'OR' => [
+            ['OR' => [
                 ['license_id' => null],
                 ['license_id' => ""]
             ]
+            ]
```

Comment:
```suggestion
            [
                'OR' => [
                    ['license_id' => null],
                    ['license_id' => ""]
                ]
            ]
```

Reply:
Was already done on my side, but I forget to push \o/ Should be fixed now, thanks :)